book-library/script.js

```diff
@@ -25,20 +25,20 @@ const author = document.getElementById("author");
 const pages = document.getElementById("pages");
 const check = document.getElementById("check");
 
-//check the right input from forms and if its ok -> add the new book (object in array)
-//via Book function and start render function
 function submit() {
   if (
     title.value == null ||
     title.value == "" ||
+    author.value == null || // Check for author input
+    author.value == "" ||
     pages.value == null ||
     pages.value == ""
   ) {
```

Comment:
Nazanin, I would suggest instead of explicitly checking for null/empty string here, use falsy check like this:

`if (!title.value || ! author.value || !pages.value) { ... }`

This way JavaScript will check for null/undefined/empty string for you.